### PR TITLE
IEA-0001  Improvements and updates to the library ImageEditor-Android

### PR DESCRIPTION
**Pull request by JamileFerreira**
This PR is intended to make some improvements and updates to the library.

**The modifications made were:**

      - Use oxeanbits/ImageEditAndroid fork             
      - Update libraries/ min and target SDK version  (graddle, JDK, etc) 
      - Create jitpack version
      - Hide Stickers and Beauty information 
      - Update EditImageActivity Title ("ImageEditAndroid => Edit image")
      - Replace "Edit" label to "Draw"              
      - Translate/Capital Letter to all texts 
           
 The demo is still included in this project for any lib usage testing.
 
 
**Project after changes:**

https://user-images.githubusercontent.com/9625556/131024211-df2d1f86-8505-4767-bb11-baa186e166a8.mp4

**Testing the library in the DPMS project**

Add the dependency

    

>  dependencies {
> 	        implementation 'com.github.JamileFerreira:ImageEditor-Android:v1.2'
> 	}

https://user-images.githubusercontent.com/9625556/131127881-b2518dde-4de1-4024-94c4-9cb12eefe4ff.mp4

Interface change:
Undo and redo button
Editing screen with all black background

Screenshot_2021-08-30-12-02-59

### CODE REVIEW

File: imageeditlibrary/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<resources>
+    <string name="app_name">Éditer l'image</string>
+    <string name="no_images">Non trouvé l'image</string>
```
Comment (rodiguif) [inline imageeditlibrary/src/main/res/values-fr/strings.xml]: compilation error :warning:
```suggestion
    <string name="app_name">Éditer l\'image</string>
    <string name="no_images">Non trouvé l\'image</string>
```

File: imageeditlibrary/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <string name="crop">Recadrer</string>
+    <string name="rotate">Faire pivoter</string>
+    <string name="text">Texte</string>
+    <string name="edit">Dessiner</string>
+
+    <string name="saving_image">Enregistrer...</string>
+    <string name="input_hint">Entrada de texto ...</string>
+
+    <string name="no_choose">Veuillez choisir une image à modifier</string>
+    <string name="save_error">Enregistrer l'erreur!</string>
```
Comment (rodiguif) [inline imageeditlibrary/src/main/res/values-fr/strings.xml]: :warning:
```suggestion
    <string name="save_error">Enregistrer l\'erreur!</string>
```

**Review by rodiguif — CHANGES_REQUESTED** [review]
Would be nice to add the `.iml` files and the entire `.idea` directory to the `gitignore`.

File: .idea/jarRepositories.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tiagogoncalves) [inline .idea/jarRepositories.xml]: Please put the whole `.idea` folder in `.gitignore`

**Comment by JamileFerreira** [comment]
> Would be nice to add the `.iml` files and the entire `.idea` directory to the `gitignore`.

@rodiguif  Besides Image Editor-Android.iml, is there more?

File: .idea/modules/imageeditlibrary/ImageEditor-Android.imageeditlibrary.iml [hunk 4]
```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tiagogoncalves) [inline .idea/modules/imageeditlibrary/ImageEditor-Android.imageeditlibrary.iml]: the `.idea` files are still here @JamileFerreira

File: imageeditlibrary/src/main/res/values-pt/strings.xml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        <item>Filme</item>
+        <item>Memória</item>
+        <item>Iogurte</item>
+        <item>Fugaz</item>
+        <item>Brilhar</item>
+        <item>Mosaico</item>
+    </string-array>
+
+    <string name="more">Mais</string>
+    <string name="exit_without_save">A imagem editada não é salva. Você quer? </string>
```
Comment (tiagogoncalves) [inline imageeditlibrary/src/main/res/values-pt/strings.xml]: Tem Certeza? A edição não será salva.

File: imageeditlibrary/src/main/res/values-pt/strings.xml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    <string name="confirm"> Confirme</string>
```
Comment (tiagogoncalves) [inline imageeditlibrary/src/main/res/values-pt/strings.xml]: Confirmar

File: imageeditlibrary/src/main/res/values-pt/strings.xml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    <string name="cancel">Cancelar</string>
+
+    <string name="beauty">Beleza</string>
+    <string name="smooth">Suave</string>
+    <string name="white_skin">Pele branca</string>
```
Comment (tiagogoncalves) [inline imageeditlibrary/src/main/res/values-pt/strings.xml]:
```suggestion
    <string name="smooth">Suave</string>
```

File: imageeditlibrary/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -1,54 +1,54 @@
...
-    <string name="cancel">cancel</string>
+    <string name="confirm">Confirm</string>
+    <string name="cancel">Cancel</string>
 
-    <string name="beauty">beauty</string>
-    <string name="smooth">smooth</string>
-    <string name="white_skin">white skin</string>
+    <string name="beauty">Ceauty</string>
+    <string name="smooth">Smooth</string>
+    <string name="white_skin">White skin</string>
```
Comment (tiagogoncalves) [inline imageeditlibrary/src/main/res/values/strings.xml]:
```suggestion
    <string name="smooth">Smooth</string>
```

**Review by tiagogoncalves — CHANGES_REQUESTED** [review]

File: imageeditlibrary/src/main/res/values/strings.xml (hunk 8 above)
Comment (JamileFerreira) [reply imageeditlibrary/src/main/res/values/strings.xml]: There are some files in the build folder that are generated and should not be edited and use these references.
Removed from translations of: de, es, fr, no, pt.

**Review by tiagogoncalves — APPROVED** [review]

**Pull request merged**